llvm/test/CodeGen/AMDGPU/amdgpu-codegenprepare-break-large-phis.ll

```diff
@@ -1197,3 +1197,54 @@ reallyfinally:
   store <5 x double> %val, ptr %out, align 1
   ret void
 }
+
+define amdgpu_kernel void @zext_i1_as_index(i1 %Bool, ptr %Ptr, <4 x float> %Vec1, <4 x float> %Vec2) {
+; OPT-LABEL: @test(
```

Comment:
Label name doesn't match. Also use the issue number

Reply:
Done